libs/verify-android-environment.sh

```diff
@@ -70,4 +68,7 @@ if [[ -z "${CI}" ]]; then
   fi
 fi
 
+echo "Installing uniffi_bindgen if missing; it's necessary for binding generation during the builds."
+cargo install uniffi_bindgen --version 0.6.1
```

Comment:
nit: please move this into `verify-common.sh` since it applies to all platforms, not just Android.

Reply:
From https://github.com/mozilla/application-services/pull/3836/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R103, we actually need `cargo install --version 0.7.0 uniffi_bindgen` now.